src/main/java/org/mifos/processor/bulk/zeebe/worker/AccountLookupWorker.java

```diff
@@ -68,16 +78,19 @@ public void setup() {
             exchange.setProperty(HOST, identityMapperURL);
             exchange.setProperty(PAYEE_IDENTITY, payeeIdentity);
             exchange.setProperty(PAYMENT_MODALITY, paymentModality);
-            exchange.setProperty(CALLBACK, bulkURL + "/accountLookup/Callback");
+            exchange.setProperty(CALLBACK, identityMapperURL + accountLookupCallback);
```

Comment:
The call back should be coming from Identity to Bulk, isn't it?

why the URL looks interchanged?

Reply:
The callback doesn't need to come back to bulk, after receiving the callback we are just resuming the workflow that can be done from either service.

Reply:
This needs to be revisited  [PHEE-460 GOV-517 - call back for account lookup should come back to bulk processor. not identity mapper.](https://fynarfin.atlassian.net/browse/PHEE-460?atlOrigin=eyJpIjoiNDlhZDYxMjM3ZTJlNDI1NWE4OWRkYjhkNGMzMDNiZGMiLCJwIjoiamlyYS1zbGFjay1pbnQifQ)